docs/index.rst

```diff
@@ -1,218 +1,111 @@
 
-   workflow/pycbc_make_psd_estimation_workflow
-   workflow/pycbc_make_coinc_search_workflow
-   workflow/pygrb.rst
-
-=============================================
-Template Banks, Hardware Injections, and more
-=============================================
-
-Users who are interested in tools that PyCBC provides for various other
-analysis tasks (e.g. template bank generation, hardware injections, and testing
-template banks) should read the documentation at:
+   tutorials
+   inference
+   apps
 
 .. toctree::
+   :caption: Dev Guides
+   :hidden:
    :maxdepth: 1
 
-   tmpltbank
-   hwinj
-   banksim
-   faithsim
-   upload_to_gracedb
-   pycbc_condition_strain
-
-=====================================
-Extending PyCBC with external plugins
-=====================================
-
-Would you like to use a waveform model that PyCBC doesn't have? Or maybe
-you have your own waveform you'd like to use for a search, parameter estimation
-, etc. PyCBC supports a plug-in archictecture for external waveform models.
+   extend
+   devs
 
 .. toctree::
+   :hidden:
    :maxdepth: 1
 
-   waveform_plugin
+.. card:: Core Library Examples
+    :link:  tutorials
+    :link-type: ref
 
-==========================================
-Library Examples and Interactive Tutorials
-==========================================
+    The core library provides the tools to do GW data analysis.
+    See examples of how to use PyCBC core tools to read gravitational-wave
+    data, query detector status, filter data, generate waveforms, estimate
+    PSDs, calculate SNRs, and much more.
 
-We have interactive tutorials and examples of using the pycbc.
-`Please give them a try! <https://github.com/gwastro/PyCBC-Tutorials>`_.
 
-In addition we have some examples below.
+.. card:: PyCBC Inference
+    :link:  inference
+    :link-type: ref
 
-.. toctree::
-   :maxdepth: 2
-
-   catalog
-   dataquality
-   frame
-   fft
-   gw150914
-   detector
-   psd
-   noise
-   waveform
-   filter
-   distributions
-
-=============================
-Documentation for Developers
-=============================
-
-Documentation on building stand-alone bundled executables with PyInstaller is available at:
+    Easy-to-use, configurable, and robust Parameter Estimation for Gravitational-wave Astronomy and Multi-messenger analysis.
 
-.. toctree::
-   :maxdepth: 1
 
-   building_bundled_executables
+.. card:: Detect Signals in Archival Data
+    :link:  search_workflow
+    :link-type: ref
 
-PyCBC developers should read the pages below which explain how to write
-documentation, develop the code, and create releases:
+    The deep analysis used to detect gravitational-wave sources in
+    archival data.
 
-.. toctree::
-   :maxdepth: 1
+.. card:: Detect Signals in association with GRBs
+    :link:  pygrb
+    :link-type: ref
 
-   documentation
-   release
+    Targetted analysis to detect gravitational-wave sources in association
+    with gamma-raty bursts and other transient sources.
```

Comment:
gamma-raty. Similarly here it might be worth saying that this is development with the aim being to have this fully implemented in PyCBC within a year.